firebase-inappmessaging/gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=22.0.1
+version=22.1.0
```

Comment:
why is this necessary?

Reply:
since its adding two new api methods to livesession which is a minor verson change